cli/import-strings.php

```diff
@@ -42,6 +42,7 @@
     'message'       => '',
     'userinfo'      => 'David Mudrak <david@moodle.com>',
     'commithash'    => null,
+    'yes'         => false,
```

Comment:
Oh lord, you tease my alignment obsession here :-) I am not a fan of these alignments any more these days (I think there should strictly be just one space around `=>` and done). But in this case, would not it be better to simply have it aligned?